Backpack/src/main/java/net/skyscanner/backpack/navbar/BpkNavBar.kt

```diff
@@ -134,20 +118,17 @@ class BpkNavBar @JvmOverloads constructor(
             R.styleable.BpkNavBar,
             defStyleAttr, 0,
         ).use {
-            expandedTitleColor = it.getColor(R.styleable.BpkNavBar_navBarExpandedTextColor, expandedTitleColor)
-            collapsedTextColor = it.getColor(R.styleable.BpkNavBar_navBarCollapsedTextColor, collapsedTextColor)
             title = it.getString(R.styleable.BpkNavBar_navBarTitle)
             navIcon = it.getDrawable(R.styleable.BpkNavBar_navBarIcon)
             menu = it.getResourceId(R.styleable.BpkNavBar_navBarMenu, menu)
         }
 
-        this.expandedTitleColor = expandedTitleColor
-        this.collapsedTitleColor = collapsedTextColor
         this.background = ColorDrawable(context.getColor(R.color.bpkCanvas))
         this.title = title
         this.icon = navIcon
         this.navAction = navAction
         this.menu = menu
+        this.stateListAnimator = shadowStateListAnimator()
```

Comment:
The shadow by default is NOT 4dp, and this is how you suppose to set the preferred shadow